### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Daniil-A4**

### CODE REVIEW

File: src/formatDate.js
Comment (maxim2310) [inline src/formatDate.js:68]: use a meaningful name for the variable

File: src/formatDate.js
Comment (maxim2310) [inline src/formatDate.js:76]: use a meaningful name for the variable

File: src/formatDate.js
Comment (maxim2310) [inline src/formatDate.js:65]: better use indexOf without loop for

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just couple tiny things needs to be fixed before approval)

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**
